sunbeam-python/sunbeam/core/juju.py

```diff
@@ -575,33 +575,39 @@ async def run_cmd_on_unit_payload(
         name: str,
         model: str,
         cmd: str,
+        env: str,
```

Comment:
I think it would be cleaner api if env was defined as `env: dict[str, str] | None = None` after container, so that if there's not env, no need to pass a value, and have the function put the dict to the correct format, don't you think?